### PR TITLE
Expand Gen AI, replace some figures with custom TiKz plots

### PR DESCRIPTION
**Pull request by chris-boson**
### after

Now, we 
- Expand on Gen AI
- Replace some borrowed images with TiKz

Still would need to generate a couple more..

### CODE REVIEW

File: book/chapters/NLP/generative_models.tex [hunk 1]
```diff
@@ -7,35 +7,35 @@ \section{Generative Models and Large Language Models}
...
-we find patterns of common-sense reasoning in responses. In the case of ChatGPT, a fine-tuning process with
-human labelers called Reinforcement Learning With Human Feedback \textit{RLHF} targeted iteratively improving the model by specifically
-rewarding high-quality content and penalizing toxic feedback. Because these models are trained on text across the internet, which
+What makes these newer language models such as OpenAI's \textit{ChatGPT}, Meta AI's \textit{Llama}\cite{touvron2023llama}, or Google's \textit{Bard} so special is their ability to generalize to new tasks \textit{without any fine-tuning or training}.
+What this means is that we can simply provide an instruction (also called a \textit{prompt}) to the model, and it will generate text that is consistent with it.
+Moreover, these models are exhibiting remarkable emergent abilities, meaning things they were never explicitly trained on.
+These include common sense reasoning, the ability to use tools (e.g. search, calculator, write and execute code, invoke APIs etc.), and even theory of mind (e.g. understanding that other agents have their own beliefs, desires, and intentions)\cite{bubeck2023sparks,openai2023gpt4}.
```
Comment (robmsylvester) [inline book/chapters/NLP/generative_models.tex]: common-sense

File: book/chapters/NLP/generative_models.tex [hunk 2]
```diff
@@ -7,35 +7,35 @@ \section{Generative Models and Large Language Models}
...
+
+As it turns out, the key to unlocking these abilities lies in an elaborate fine-tuning process after extensive pre-training on a large corpus of text.
+In the case of ChatGPT, this was done in a novel way using a method called Reinforcement Learning with Human Feedback \textit{RLHF} targeted iteratively improving the model by specifically
```
Comment (robmsylvester) [inline book/chapters/NLP/generative_models.tex]: This sentence is missing a word. Maybe “which targets…”

**Review by robmsylvester — APPROVED** [review]
Niiice

**Comment by chris-boson** [comment]
Still a couple more plots to replace but this should bring it pretty close

**Pull request merged**